common/views/components/IIIFResponsiveImage/IIIFResponsiveImage.js

```diff
@@ -13,8 +13,9 @@ type Props = {|
   lang: ?string,
   isLazy: boolean,
   clickHandler?: () => void | Promise<void>,
-  loadHandler?: () => void | Promise<void>,
+  onLoadHandler?: () => void | Promise<void>,
```

Comment:
I was wondering about handler names the other day. If `load` is the event and `onLoad` is where you define your handler for that event, doesn't `loadHandler` make more sense?

Reply:
you're right, I keep confusing myself with what to call these. will change